playbooks/common/openshift-cluster/upgrades/pre/verify_upgrade_targets.yml

```diff
@@ -10,40 +10,31 @@
     tasks_from: registry_auth.yml
   when: oreg_auth_user is defined
 
-- name: Verify containers are available for upgrade
-  command: >
-    docker pull {{ openshift_cli_image }}
-  register: pull_result
-  changed_when: "'Downloaded newer image' in pull_result.stdout"
-  when: openshift_is_containerized | bool
+- name: Check latest available OpenShift RPM version
```

Comment:
In many places, openshift_is_containerized is also a stand-in for openshift_is_atomic.

We need to ensure atomic is accounted for in this patch set.

Reply:
Right, I'll mark this as WIP and ensure it works correctly with Atomic

Reply:
is repoquery valid on atomic?